### PR TITLE
Fix: Add className to IconText and Menu

### PR DESCRIPTION
**Pull request by nicosampler**

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr28--safereactcomponents.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr28--safereactcomponents.review.gnosisdev.com

File: src/navigation/Menu/index.tsx [hunk 1]
```diff
@@ -14,8 +13,11 @@ const Wrapper = styled.div`
 
 type Props = {
   children: any;
```
Comment (mikheevm) [inline src/navigation/Menu/index.tsx:15]: please remove any

File: src/navigation/Menu/index.tsx [hunk 2]
```diff
@@ -14,8 +13,11 @@ const Wrapper = styled.div`
...
+  className?: string;
 };
 
-const Menu = ({ children }: Props) => <Wrapper>{children}</Wrapper>;
+const Menu = ({ children, className }: Props) => (
```
Comment (mikheevm) [inline src/navigation/Menu/index.tsx:19]: no return type

File: src/dataDisplay/IconText/index.tsx [hunk 3]
```diff
@@ -25,11 +26,20 @@ const StyledIconText = styled.div`
 /**
  * The `IconText` renders an icon next to a text
  */
-function IconText({ iconSize, textSize, iconType, text, color }: Props) {
+function IconText({
```
Comment (mikheevm) [inline src/dataDisplay/IconText/index.tsx:29]: no return type, mixed usage of function/arrow function syntax

File: src/navigation/Menu/index.tsx (hunk 2 above)
Comment (nicosampler) [reply src/navigation/Menu/index.tsx]: done!

File: src/dataDisplay/IconText/index.tsx (hunk 3 above)
Comment (nicosampler) [reply src/dataDisplay/IconText/index.tsx]: done!

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr28--safereactcomponents.review.gnosisdev.com

File: src/navigation/Menu/index.tsx (hunk 1 above)
Comment (nicosampler) [reply src/navigation/Menu/index.tsx]: done!

**Comment by nicosampler** [comment]
@mikheevm are we good to go here?

**Pull request merged**